crates/starknet-types-core/src/hash/pedersen.rs

```diff
@@ -0,0 +1,68 @@
+use crate::felt::Felt;
+use lambdaworks_crypto::hash::pedersen::Pedersen as PedersenLambdaworks;
+use lambdaworks_math::field::{
+    element::FieldElement, fields::fft_friendly::stark_252_prime_field::Stark252PrimeField,
+};
+use num_traits::FromPrimitive;
+
+use super::traits::StarkHash;
+
+pub struct Pedersen;
+
+impl StarkHash for Pedersen {
+    /// Computes the Pedersen hash of two Felts, as defined
+    /// in <https://docs.starknet.io/documentation/architecture_and_concepts/Hashing/hash-functions/#pedersen_hash.>
+    fn hash(felt_0: &Felt, felt_1: &Felt) -> Felt {
+        let pedersen = PedersenLambdaworks::default();
+
+        let hash = pedersen.hash(&felt_0.0, &felt_1.0);
+
+        Felt(hash)
+    }
+
+    /// Computes the Pedersen hash of an array of Felts, as defined
+    /// in <https://docs.starknet.io/documentation/architecture_and_concepts/Hashing/hash-functions/#array_hashing.>
+    fn hash_array(felts: &[Felt]) -> Felt {
+        let pedersen = PedersenLambdaworks::default();
+        let data_len =
+            Felt::from_u128(u128::try_from(felts.len()).expect("Got 2^128 felts or more."))
+                .unwrap();
```

Comment:
You can use the infallible `from` here:
```suggestion
            Felt::from(felts.len());
```

Reply:
Thanks!, nice one
Done [3363c1b](https://github.com/starknet-io/types-rs/pull/12/commits/3363c1bcd55f3bb30418ea6bf1d9385f45e23944)